src/interface/params.cpp

```diff
@@ -109,6 +110,7 @@ void Params::ReadFromRestart(const std::string &prefix, const HDF5::H5G &group)
   // strings
   ReadFromHDF5AllParamsOfType<std::string>(prefix, group);
   ReadFromHDF5AllParamsOfType<std::vector<std::string>>(prefix, group);
+  ReadFromHDF5AllParamsOfType<std::vector<char>>(prefix, group);
```

Comment:
This lets us read binary files into `Params` and keep them in `.rhdf` output files.